src/api/mutations/index.ts

```diff
@@ -1,3 +1,6 @@
 export * from './useLogin';
 export * from './useMailVerification';
 export * from './useRegister';
+// ProfilePage
```

Comment:
주석 없이 문단을 구별하는 것만으로도 도메인이 구분되지 않을까요?! 필요하다고 느끼시면 4번째 줄을 더 띄워야 할 것 같아요.

```suggestion

// ProfilePage
```

Reply:
문단으로도 충분할 것 같습니다! 주석 지우고 올릴게요!